ArchiSteamFarm/IPC/Responses/GitHubReleaseResponse.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ArchiSteamFarm.IPC.Responses {
+	public sealed class GitHubReleaseResponse {
+		private GitHub.ReleaseResponse ReleaseResponse;
+
+		[JsonProperty]
+		private IEnumerable<string> Changes => ReleaseResponse.MarkdownBody.Split('\n').Where(line => line.StartsWith("- ")).Select(change => ArchiSteamFarm.Utilities.MarkdownToText(change)).Where(change => !string.IsNullOrEmpty(change));
```

Comment:
This is unacceptable, you should not provide `IEnumerable` of changes, but just pure body, and in HTML, not plain text.